snuba/datasets/plans/single_storage.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional, Sequence
+
+from snuba.datasets.plans.query_plan import (
+    QueryPlanExecutionStrategy,
+    QueryRunner,
+    StorageQueryPlan,
+    StorageQueryPlanBuilder,
+)
+from snuba.datasets.storage import QueryStorageSelector, ReadableStorage
+
+# TODO: Importing snuba.web here is just wrong. What's need to be done to avoid this
+# dependency is a refactoring of the methods that return RawQueryResult to make them
+# depend on Result + some debug data structure instead. Also It requires removing
+# extra data from the result of the query.
+from snuba.web import RawQueryResult
+from snuba.query.query_processor import QueryProcessor
+from snuba.request import Request
+
+
+class SimpleQueryPlanExecutionStrategy(QueryPlanExecutionStrategy):
+    def execute(self, request: Request, runner: QueryRunner) -> RawQueryResult:
+        return runner(request)
```

Comment:
This doesn't do much yet. What are the planned future responsibilities of QueryPlanExecutionStrategy? Where will splitting be?

Reply:
splitting will be in another class that extends QUeryPlanExecutionStrategy.
see #831